### DISCUSSION TITLE
[Idea] Add glpi agent in the winget library

### DISCUSSION THREAD

**Discussion by tomamplius** [glpi-project/glpi-agent#626] [Ideas]
### Your idea

Add glpi agent in the winget librairy 

https://learn.microsoft.com/fr-fr/windows/package-manager/winget/

It's will make easy to install and upgrade glpi on windows computers

**Comment by g-bougard** [comment]
Hi @tomamplius 

this is a good idea, and even supporting inventory softwares installed this way could be interesting if not seen by the standard software inventory process.

I'll investigate before 1.8 release.

**Comment by clementameline** [comment]
Hi,
This would be really useful given the recent vulenrabilities ([https://glpi-project.org/fr/agent-glpi-version-1-7-2-disponible/](https://glpi-project.org/fr/agent-glpi-version-1-7-2-disponible/)).

**Comment by g-bougard** [comment]
Just for your information, I started the process for the latest release yesterday:
https://github.com/microsoft/winget-pkgs/pull/148378

It was merged and today, you can see it in the repo:
```
C:\Users\gbougard>winget search glpi
Nom        ID                      Version Correspondance Source
----------------------------------------------------------------
GLPI Agent GLPI-Project.GLPI-Agent 1.7.3   Tag: glpi      winget

C:\Users\gbougard>winget search glpi-agent
Nom        ID                      Version Correspondance      Source
---------------------------------------------------------------------
GLPI Agent GLPI-Project.GLPI-Agent 1.7.3   Moniker: glpi-agent winget

```

**Comment by g-bougard** [comment]
I also just added a dedicated chapter in documentation:
https://glpi-agent.readthedocs.io/en/latest/installation/index.html#winget
